### ISSUE TITLE
Issue with HIPPO weights 

### ISSUE THREAD

**Issue by SalimMegat** [LieberInstitute/brainseq_phase2#34]
HI, 

I have noticed that an issue in the HIPPO weights as the "blump" models contain NA values probably because there was a typo in the script. Indeed, the NA values are following the "blump" model while it should be "blup". Do you think that would be possible for you to share an uploaded version of the HIPPO weights if available ? 

Again, many thanks for your help ! 

Best, 

Salim.

**Comment by SalimMegat** [comment]
*updated

**Comment by lcolladotor** [comment]
Hi,

Apologies for the late reply @SalimMegat. I didn't see this earlier (I'm going through https://lcolladotor.github.io/pkgs/ and noticing issues like this one >.<).

We won't be able to run `blump` given that it takes a lot of time to run, such that it becomes computationally implausible to run it at the exon and exon-exon junction expression levels.

We also noticed that typo ourselves, and well, I did try seeing if we could fix the weights but even a small test with 20 genes took over 30 min to run. So say, 400,000 exons would take over 600k minutes (10k hours, 416 days).

Best,
Leo
